styles/desktop.css

```diff
@@ -7,6 +7,16 @@
     margin: 16px 0 16px 143px;
   }
 
+  @keyframes rotation {
+    from {
+      transform: rotate(0deg);
+    }
+
+    to {
+      transform: rotate(360deg);
+    }
+  }
+
   .icon-menu {
     display: none;
   }
```

Comment:
😄  on the desktop version of your project, your hamburger menu is visible please fix this and try again